Week 8/scheduler/src/Scheduler.java

```diff
@@ -0,0 +1,95 @@
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.util.PriorityQueue;
+
+// Schedule the jobs in the given file and prints the log.
+// The scheduler selects tasks using priority queue.
+// Information on job priority is contained in Job class.
```

Comment:
Typos around every comment
it should be
Schedule the jobs in the given file and print the logs.
The scheduler selects tasks using a priority queue.
Information on job priority is contained in Job class.